File: app/src/main/java/vfunny/shortvideovfunnyapp/Login/data/Post.java
package vfunny.shortvideovfunnyapp.Login.data;

import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;

import org.jetbrains.annotations.Nullable;

/**
 * Created on 02/05/2017.
 * Copyright by 01eg.me
 */

public class Post {

    private Integer likes;
    private Integer comments;
    private String image;
    private String video;
    private String user;
    public String key;

    public Post() {
    }

    public Integer getLikes() {
        return likes;
    }

    public void setLikes(Integer likes) {
        this.likes = likes;
    }

    public Integer getComments() {
        return comments;
    }

    public void setComments(Integer comments) {
        this.comments = comments;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getVideo() {
        return video;
    }

    public void setVideo(String video) {
        this.video = video;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }
}

File: app/src/main/java/vfunny/shortvideovfunnyapp/Login/data/Post.kt
package vfunny.shortvideovfunnyapp.Login.data

import com.google.firebase.Timestamp

/**
 * Created on 02/05/2019.
 * Copyright by shresthasaurabh86@gmail.com
 */
class Post {
    var image: String? = null
    var video: String? = null
    var key: String? = null
    var timestamp: Any? = null

    override fun equals(other: Any?): Boolean {
        if (other is Post) {
            return image == other.image &&
                    video == other.video &&
                    timestamp == other.timestamp
        }
        return false
    }

}